src/promo/DonationDialog.jsx

```diff
@@ -79,14 +106,14 @@ export const DonationDialog = ({
     open,
     onClose,
     onEnter: onVisible,
-    scroll: 'body',
```

Comment:
Sorry I didn't get a chance to review this before it was merged, but the reason we were using `scroll: 'body'` is to work around this issue:

![Screenshot at 2019-05-14 16-52-44](https://user-images.githubusercontent.com/2295892/57676834-cb3d9d80-7668-11e9-8b3d-1251527c14e4.png)

I think it's a much better user experience if they can scroll the whole page to see the dialog contents (which is also the existing behaviour of our "legacy" lightboxes in TC instead of having an internal scroll.

Reply:
Thanks for catching that, I will fix this in a follow-up PR.